ote/src/cljs/ote/views/front_page.cljs

```diff
@@ -101,13 +101,19 @@
    [:div.row
     [:div {:class "col-md-12"}
      [:h1 (tr [:common-texts :own-api-list])
+
+
       [ui/raised-button {:label (tr [:buttons :add-transport-service])
                         :style {:float "right"}
                         :on-click #(do
                                      (.preventDefault %)
                                      (e! (ts/->OpenTransportServiceTypePage)))
                         :primary  true
-                        :icon (ic/content-add)}]]]
+                        :icon (ic/content-add)}]]
+     [:div {:style {:border "red 4px dashed"}}
+      [:p {:style {:padding "10px"}} "HUOM: Tämä on NAP -palvelun testiversio."
+       [:br]
+       "Tiedot eivät siirry varsinaiseen NAP-palveluun, joka avataan tuotannossa 20.12.2017."]]]
```

Comment:
This needs to made conditional as it must not be shown in the production environment.

Reply:
We could pass dev (or is staging currectly) / production param, and use that in the conditional.